packages/webapp/src/util/task.ts

```diff
@@ -61,39 +58,29 @@ type FormSoilAmendmentTaskProduct = {
 };
 
 type DBSoilAmendmentTask = {
-  furrow_hole_depth?: number;
-  furrow_hole_depth_unit?: string;
-  other_application_method?: string;
-  [key: string]: any;
-};
-
-type FormSoilAmendmentTask = {
-  furrow_hole_depth?: number;
-  furrow_hole_depth_unit?: UnitOption;
-  other_application_method?: string;
-  [key: string]: any;
-};
-
-type DBTask = {
   soil_amendment_task_products: DBSoilAmendmentTaskProduct[];
   [key: string]: any;
 };
 
-type FormTask = {
```

Comment:
I don't know if I would have undid the updated naming since `soil_amendment_task` has meaning and does not have property of `soil_amendment_task_products`.